llvm/include/llvm/DebugInfo/DWARF/DWARFUnit.h

```diff
@@ -224,7 +224,7 @@ class DWARFUnit {
   const DWARFSection *AddrOffsetSection;
   DWARFUnit *SU;
   Optional<uint64_t> AddrOffsetSectionBase;
-  bool isLittleEndian;
```

Comment:
I know this is not following the LLVM code style, but this is creating a huge diff in many upstream files that we should avoid (it also would make it easier to review without this diff)

Reply:
The correct thing to do here would be to make the change upstream and then rebase this patch.

Reply:
It has been merged with 636de2bf342e